visualizations/frontend/other/TTable.vue

```diff
@@ -380,6 +376,38 @@ export default {
     uniqueOriginalRows() {
       return _.uniq(this.internalRows.map((r) => r.originalRow));
     },
+    layerRows() {
+      if (typeof this.rows === "function")
+        return this.rows(this.$attrs?.["t-layer"]);
+      if (this.rows) return this.rows;
+      return [];
+    },
+  },
+  watch: {
+    loading() {
```

Comment:
There is already a callback for this case, you can see it here: https://github.com/snyk/topcoat-core/blob/main/frontend/src/mixins/visualization_mixin.js#L425-L432

Basically, we should be able to move all the code inside the `if (!this.loading)` into the `onVisualizationUpdated` method.

Reply:
Sweet! That is way simpler!